app/src/main/kotlin/com/wire/android/ui/common/CopyButton.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.android.ui.common
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.button.IconAlignment
+import com.wire.android.ui.common.button.WireTertiaryButton
+import com.wire.android.ui.common.button.wireSecondaryButtonColors
+
+@Composable
+fun CopyButton(onCopyClicked: () -> Unit, modifier: Modifier = Modifier) {
+    WireTertiaryButton(
+        colors = wireSecondaryButtonColors(),
```

Comment:
TertiaryButton with SecondaryButtonColors? Shouldn't we just use SecondaryButton then?